packages/react/src/components/form/Input/Input.mdx

```diff
@@ -11,69 +11,40 @@ import * as InputStories from './Input.stories';
 
 # Input
 
-`Input` kalles tekstfelt på norsk. Det er et inndatafelt som for eksempel gir brukerne mulighet til å skrive korte tekster eller tall.
+`Input` er et skjemaelement for å samle inn brukerdata. Det tilbyr grunnleggende funksjonalitet og krever tilpasning med styling og script. `Input` er ideell når du trenger full kontroll over komponentenes oppsett og validering, for eksempel ved bygging av spesialtilpassede elementer.
```

Comment:
"krever tilpasning med styling og script" - Denne forstår eg ikkje, kan du utdjupe?
Ein `<input />` funker fint uten at du gjer noko

Reply:
Da har jeg skrevet feil. Jeg tenkte mer på at den kan tilpasses med styling og script for spesifikke behov. Jeg fjerner den delen 😊

Reply:
Skjønner, synst det er fint det som står no :)